js/state/frameStateUtil.js

```diff
@@ -480,7 +480,7 @@ const isFirstFrameKeyInTabPage = (state, frameKey) => {
   const firstFrame = unpinnedTabs
     .slice(startingFrameIndex, startingFrameIndex + tabsPerTabPage).first()
 
-  return firstFrame.get('key') === frameKey
+  return !!firstFrame && firstFrame.get('key') === frameKey
```

Comment:
do we need to have `!!` if I understand correctly `firstFrame` is (if exists) map, so I think that we should be ok without double negation

Reply:
ya it's was me forcing coercion. I tried without it w/ tests and worked same way so I made your request